core/jazz_create-serverless-service/index.js

```diff
@@ -357,10 +359,49 @@ var validateEventName = (eventType, sourceName, config) => {
   }
 };
 
+var updateAclPolicy = (serviceId, authToken, user_id, permission, category, config) => {
+    return new Promise((resolve, reject) => {
+        let svcPayload = {
+            uri: config.SERVICE_API_URL + config.ACL_ENDPOINT,
+            method: 'POST',
+            headers: { 'Authorization': authToken },
+            json: [
+                {
+                    "serviceId": serviceId,
+                    "policies": [
+                        {
+                            "userId": user_id,
+                            "permission": permission,
+                            "category": category
+                        }
+                    ]
+                }
+            ],
+            rejectUnauthorized: false
+        };
+
+        request(svcPayload, (error, response, body) => {
+            if(error) {
+                logger.error(`Error while updating policies using ACL: ${error}`);
+                reject(error);
+            } else {
+                logger.debug(`ACL response: ${JSON.stringify(response)}`);
+                if(body && body.data && body.data.success) {
```

Comment:
Make sure the response contains "success"

Reply:
Updated